src/main/java/fr/lunatech/timekeeper/services/responses/EventTemplateResponse.java

```diff
@@ -51,15 +51,15 @@
     private final LocalDateTime endDateTime;
 
     @Null
-    private final List<EventTemplateResponse.Attendee> attendees;
+    private final List<User.Attendee> attendees;
```

Comment:
For me, Attendee is some sort of DTO and thus, should be part of EventTemplateResponse. Not part of the model.

Reply:
yes of course, I move it into Response package